<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatcherInputDialog</class>
 <widget class="QDialog" name="WatcherInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="siteLable">
       <property name="text">
        <string>Web page</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="siteEdit">
       <property name="toolTip">
        <string>Url of web page, which will be loaded during updation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="queryLabel">
       <property name="text">
        <string>JavaScript query</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="queryEdit">
       <property name="toolTip">
        <string>JavaScript code, which will executed on the loaded page. Result of code will be converted to string and will be added to results table</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="intervalLabel">
       <property name="text">
        <string>Update Interval</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="intervalEdit"/>
       </item>
       <item>
        <widget class="QComboBox" name="intervalComboBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title (optional)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleEdit">
       <property name="toolTip">
        <string>Visible name of entry in left part of window. If the field is empty, then the name will be setted from webpage name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>siteEdit</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>queryEdit</tabstop>
  <tabstop>intervalEdit</tabstop>
  <tabstop>intervalComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WatcherInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WatcherInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
